### ISSUE TITLE
[BUG] Android - Label in Popup goes out of view instead of wrapping  (with Maui.Controls 8.0.7)

### ISSUE THREAD

**Issue by ramonB1996** [CommunityToolkit/Maui#1717]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

Since updating to Maui.Controls 8.0.7 and using the MCT-popup, labels flow out of the popup and do not wrap to a new line. 

This happens only when a `Button` exists in the same popup. Other controls do not cause issues for the label.

Note that this only happens in Popups. Outside of a popup, this same scenario works fine.

### Expected Behavior

Label should just use multi-lines and not stay single line. The behavior was okay in Maui.Controls 8.0.6.

### Steps To Reproduce

1. Open and run the reproduction repository
2. Click the button in the center to open the popup
3. See that the label overflows in the popup

Optional 4: Downgrade to Maui.Controls version 8.0.6 and see that the bug has disappeared

### Link to public reproduction project repository

https://github.com/ramonB1996/MCTAndroidPopupLabelIssue

### Environment

```markdown
- .NET MAUI CommunityToolkit: 7.0.1
- OS: Android API 34
- .NET MAUI: 8.0.7(SR2)
```

### Anything else?

The issue possibly occurred from this PR: https://github.com/dotnet/maui/pull/14918

But we probably have to change something about this project, rather than changing Maui.Controls to get it working. As it is still working in normal layouts.

**Comment by bijington** [comment]
I wonder if https://github.com/CommunityToolkit/Maui/pull/1683 helps

**Comment by ramonB1996** [comment]
@bijington Let's hope so :) I am willing to test this, once a new version will come out!

**Comment by BBDevelopmentStudio** [comment]
I can confirm the bug. I've been unable to find a workaround thus far. Has anyone found a viable workaround?
- apart form ditching the popups all together and push a new page onto the stack instead.

**Comment by WilliamWatterson86** [comment]
Devops seem to have updated their pipelines to use 8.0.7 so all my apps are now displaying this issue too.  Any workaround for this?

**Comment by ramonB1996** [comment]
Service release 3 (V8.0.10) for Maui sadly still seems to display this issue

**Comment by bijington** [comment]
> Service release 3 (V8.0.10) for Maui sadly still seems to display this issue
> 
> 
> 
> Edit: 
> 
> Someone said that popups seem to work for him when using this implementation: https://redth.codes/popups-with-net-maui-no-plugin-nuget-needed
> 
> 
> 
> It costs some time for implementation though, so this would be some work. Not great if you want to stick with this package.

I believe Brandon is planning on merging the PR that I referred to earlier on fairly soon. I'm still planning on building many more popup samples to verify that it behaves as expected in many different scenarios. 

There are some things that people can try to assist here:
- grab the build from PR #1683 and try running against your use case
- consider submitting a sample of your use case to our sample application 
- configure your pipeline builds to use an earlier version of .NET MAUI

**Comment by ramonB1996** [comment]
@bijington I appreciate all the effort you put into the toolkit and the extra samples that you will build for this!

Good news!
I grabbed the PR that you mentioned and used that as the reference of my sample (that I submitted with this issue) and it now seems to work as intended! I checked it against SR3 of Maui (V8.0.10). Hopefully the PR can get merged soon! 

Until this is merged, Version 8.0.6 (SR1) for Maui in combination with CommunityToolkit V7.0.1 are the latest stable release where the popups work.

Please let me know if there is anything else I can do to help

**Comment by ramonB1996** [comment]
I can hereby confirm that the sample for this bug seems to work again with the following packages: 

- CommunityToolkit.Maui version 8.0.0
- Maui.Controls: 8.0.14
- Maui.Controls.Compatibility: 8.0.14

I will close this issue because of this. Thank you for the good work :) 

If there are any questions, let me know.
